resources.md

```diff
@@ -216,6 +216,24 @@ management, security configuration, and much more. Watch one
 of the demo videos and find details in the [comprehensive user
 documentation](https://docs.starburst.io/starburst-galaxy/index.html).
 
+</div>
+
+<div markdown="1" class="item">
+
+#### [Varada Control Center](https://varada.io/vcc/)
+
+Varada is the only solution that provides dynamic big data indexing for Trino,
+delivering performance improvements and radically reducing costs. 
+It enables data data teams to seamlessly accelerate and optimize Trino workloads
+to meet specific performance and cost requirements, with zero data-ops and effective resource utilization. 
+
+The VCC (Varada Control Center) is a central hub to deploy, manage and monitor Varada clusters. 
```

Comment:
The Varada Control Center (VCC)  ... manage, and monitor 

also .. Trino clusters ?

Reply:
Makes sense. Thanks for clarifying.

Please dont forget to fix acronym and comma usage here and below

Reply:
Done